resource-managers/kubernetes/core/pom.xml

```diff
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.spark</groupId>
     <artifactId>spark-parent_2.11</artifactId>
-    <version>2.2.0-SNAPSHOT</version>
+    <version>2.1.0</version>
```

Comment:
I am not sure. The underlying version of spark is still 2.1.0, which is why I thought that was appropriate.